solutions/USACO dec15-speeding-ticket.mdx

```diff
@@ -0,0 +1,124 @@
+---
+id: usaco-568
+title: USACO 2015 December Bronze - Speeding
+author: Jesse Choe
+---
+
+[Official Analysis]: http://www.usaco.org/current/data/sol_speeding_bronze_dec15.html
+
+<LanguageSection>
+<CPPSection>
+
+## C++ Implementation
+
+```cpp
+// Created by Jesse Choe - Bronze Template
+
+#include <bits/stdc++.h>
+using namespace std;
+
+// Type aliases
+
+using ll = long long;
+using str = string; 
+using vi = vector<int>;
+using pi = pair<int,int>;
+using vpi = vector<pi>;
+using si = set<int>;
+
+// Vector Operations
+
+#define sz (int)(x).size()
+#define pb push_back
+#define all(x) begin(x), end(x)
+#define sor(x) sort(all(x))
+#define rev(x) reverse(all(x))
+#define del(x, i) erase(begin(x)+i)
+#define rem(x,i) erase(begin(x), begin(x)+i)
+
+// Pairs
+
+#define mp make_pair
+#define sec second
+#define fir first
+
+// Sets and Maps
+
+#define ins insert
+#define ez erase
+#define cnt count
+
+// Math
+
+#define MOD 1e9+7
+#define MAX_INT 1e18*9
+
+// Permutation
+
+#define possibilities(x) while(next_permutation(all(x))
+
+// Loops
+
+#define Trav(a,x) for (auto& a: x)
+#define For(i,a,b) for (int i = (a); i < (b); ++i)
```

Comment:
Can we remove / shorten these templates, or make the consistent with https://usaco.guide/general/code-con#c ?

Reply:
I think this is fine :P